### ISSUE TITLE
[question] How to add tone context to pre-training data like Gemini 2.5?

### ISSUE THREAD

**Issue by CypherpunkSamurai** [canopyai/Orpheus-TTS-0.1#245]
# Description

Hello. I'm trying to pretrain a Transformer TTS model using the provided pretrain script in the repository. I'm interested how I can possibly add tone information to the training data like Google Gemini 2.5

**Comment by amuvarma13** [comment]
You can add to it the text part of the model where the transcript is - i.e. <happy whisper> transcript: audio_tokens

**Comment by CypherpunkSamurai** [comment]
I'm trying to pretrain it from scratch would it be possible to pretrain on only unlabelled audio corpus and then fine-tune to a usable tts model? Like llms

If so how should I structure the files to use the pretrain scripts.

**Comment by amuvarma13** [comment]
It depends. 

If unlabelled = audio + transcripts (without tone information) then yes this is very similar to v0.1
If unlabelled = just audio then you will need a lot more finetuning data

**Comment by amuvarma13** [comment]
Speech gpt did unlabelled data then labelled data in stages but you can see they needed quite a bit to do text<=>audio modality alignment.

**Comment by CypherpunkSamurai** [comment]
so even if the labels are incorrect should i include them? or it would make it bad.

which is appropriate? especially for consumer gpu?

1. pretrain on unlabeled data. fine-tune on labeled high-quality data.
2. pertrain on partially-labeled data. fine-tune on labeled high-quality data.
